imagersite/imagerprofile/tests.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
+from .models import ImagerProfile
+import factory
+from . import models
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = models.User
+    username = 'super_cool_guy_foreva'
+
+
+class UserTests(TestCase):
+    def setUp(self):
+        self.user = UserFactory.create()
+
+    def test_has_profile(self):
+        assert self.user.profile in ImagerProfile.objects.all()
+
+    def test_is_active(self):
+        assert self.user.is_active
```

Comment:
You might at this point test that the profile for a user who is active is also active, that the profile for a user who is not active is not.  Test that the `ImagerProfile.active` manager returns only those profiles that belong to active users.  lots of space for more thorough and insightful tests.